docs/source/demos/lead_scoring.ipynb

```diff
@@ -126,7 +126,7 @@
    "source": [
     "X_train, X_holdout, y_train, y_holdout = evalml.preprocessing.split_data(X, y, problem_type='binary', test_size=0.2, random_seed=0)\n",
     "\n",
-    "print(X.types)"
+    "print(X.ww)"
```

Comment:
nit, but do we need to print here when we just call `X.ww` in other places?

Reply:
Just cause that's what it was like before hehe but removing the print here.